tests/monotouch-test/AVFoundation/AVAudioFormatTest.cs

```diff
@@ -0,0 +1,49 @@
+﻿// Unit test for AVAudioFormat
+// Authors: 
+// 		Whitney Schmidt (whschm@microsoft.com)
+// Copyright 2020 Microsoft Corp.
+
+using System;
+using Foundation;
+using AVFoundation;
+using NUnit.Framework;
+namespace MonoTouchFixtures.AVFoundation {
+
+	[TestFixture]
+	[Preserve (AllMembers = true)]
+	public class AVAudioFormatTest {
+
+		AVAudioFormat nullFormat = null;
+
+		[Test]
+		public void TestEqualOperatorSameInstace ()
+		{
+			using (var format = new AVAudioFormat ())
```

Comment:
Not sure whether these `using` are needed - could `format` be a field like `nullformat`?

Reply:
It's not very important (in tests) but it's good to dispose asap
if there's a disposal issue you'll know it sooner, likely with a reference to the test (stack trace)
otherwise it can happen at GC time and tracking those are much less fun :)